locations/spiders/despar.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+import re
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+class DesparSpider(scrapy.Spider):
+    #download_delay = 0.2
+    name = "despar"
+    item_attributes = {'brand': "Despar"}
+    allowed_domains = ["despar.com"]
+
+    start_urls = ([
+        'https://www.despar.com'
+    ])
```

Comment:
This doesn't need parens around the list:

```suggestion
    start_urls = [
        'https://www.despar.com'
    ]
```